public/components/marketing-pricings/2.html

```diff
@@ -0,0 +1,148 @@
+<div class="mx-auto max-w-4xl px-4 py-8 sm:px-6 sm:py-12 lg:px-8">
+    <div class="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3 sm:items-center md:gap-8">
+        <div class="border-2 rounded bg-white shadow">
+            <div class="p-6">
+                <h1 class="text-xl font-medium text-gray-700 capitalize lg:text-2xl">Basic</h1>
+                <p class="mt-4 text-gray-500">
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ex quibusdam optio velit!
+                </p>
+                <h2 class="mt-4 text-2xl font-semibold text-gray-700 sm:text-3xl">$300 <span class="text-base font-medium">/Month</span></h2>
+                <p class="mt-1 text-gray-500"> Yearly Payment</p>
+                <button class="w-full px-4 py-2 mt-6 tracking-wide text-white capitalize transition-colors duration-300 transform bg-blue-600 rounded-md hover:bg-blue-500 focus:outline-none focus:bg-blue-500 focus:ring focus:ring-blue-300 focus:ring-opacity-80"> Start Now</button>
+            </div>
+            <hr class="border-gray-200" />
+            <div class="p-6">
+                <h1 class="text-lg font-medium text-gray-700 capitalize lg:text-xl">What's included: </h1>
+                <div class="mt-8 space-y-4">
```

Comment:
`mt-8` is a little large on mobile, maybe just make it `mt-6` for all breakpoints.